universite-du-quebec-a-montreal-prenoms.csl

```diff
@@ -67,91 +67,87 @@
     <choose>
       <if type="article-journal book chapter paper-conference entry-encyclopedia entry-dictionary" match="none">
         <names variable="editor" delimiter=", " prefix=" (" suffix=")">
-          <name and="symbol" initialize="false"/>
+          <name delimiter-precedes-last="never" initialize="false" name-as-sort-order="all"/>
```

Comment:
I'm not a huge fan of adding things like `delimiter-precedes-last` into the `style` element, but since you're doing this, why are you adding it back here? Mixing and duplicating where `name` formatting is determined seems maximally confusing